### PR TITLE
Bookstore: create and remove books

### PR DESCRIPTION
**Pull request by lily-coder**
In this step I have configured my Redux Store and written actions and reducer for adding and removing books.
I have also added some UI elements to my components and made them dispatch actions.

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        category: 'economy',
+      },
+      bookProgress: {
+        chapter: null,
+        chapterName: 'Introduction',
+        completed: 0,
+      },
+    },
+  ],
+};
```
Comment (helman101) [inline src/redux/books/books.js]: - [x] Your initial state should be an empty `Array` as the project requirements said, please delete these book objects 🚀 
    - A list of default books is no longer required for this milestone 😄
    imagen

File: src/components/Bookform.js [hunk 2]
```diff
@@ -1,23 +1,53 @@
 /* eslint no-unused-vars: 0 no-undef: 0 */
-import React from 'react';
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { addBook } from '../redux/books/books';
 
 const BookForm = () => {
```
Comment (helman101) [inline src/components/Bookform.js:7]: - [x] This component should have a local state, I would recommend using `useState()` with `title` and `author`, they should have their inputs that updates the local state every time they update (`onChange`) 😸 

    - Remember to follow all the mandatory project requirements 💯 
    image

**Review by helman101 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

Hi @lily-coder  👋, 

Nice work on this milestone of the project, I really like how your project looks 😄, but there are still some changes to do.

## To Highlight 🌟 

- You did an excellent job with the different components 😄 
- I like that you even complete the optional requirement 🚀 

## Required Changes ♻️

- [x] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

gif

If anything is not 100% clear please leave your questions or comments in the PR thread, we will be glad to help you 😄.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Bookform.js (hunk 2 above)
Comment (lily-coder) [reply src/components/Bookform.js:7]: I have implemented the local state using useState with title and author. So they are updating the local state onchange

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## Thank you for implementing all the project requirements, you've done a great job :trophy:  . There is nothing else to say other than... it's time to merge it :champagne:  :tada: Congratulations

Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**